### ISSUE TITLE
Maintained?

### ISSUE THREAD

**Issue by confuser** [ftbastler/BoNeMEAL#117]
Currently re-designing the banmanager website and was considering adding a section of community based addons/apps around the BanManager ecosystem.

Is this still being actively maintained? I'd like to link to BoNeMEAL if it is.

Additionally I can see v7 database changes caused some issues, apologies for this. I've started putting together a [v8 roadmap](https://github.com/BanManagement/BanManager/issues/905) to hopefully mitigate blind sighting projects such as this. What would be the best way to communicate schema changes in future?

**Comment by oranges13** [comment]
Awesome! Thank you for the consideration. Yes this is actively maintained but if possible I was hoping to move it to my fork to maintain better control over the repository settings and CI going forward -- it's over here [www.github.com/oranges13/BoNeMEAL](http://www.github.com/oranges13/BoNeMEAL) and I'm actively developing a new version while maintaining bug fixes here.

Questions, please let me know!

**Comment by confuser** [comment]
Awesome, glad to hear. If you need any support/questions, give me a shout; BanManager's Discord https://discord.gg/59bsgZB Also open to moving the fork to the BM org if that would help?

**Comment by oranges13** [comment]
Yes! I would not mind starting from a fresh fork in the official org. That would actually be ideal.
